source/reflect.h2

```diff
@@ -1992,9 +1992,9 @@ parse_context: type =
 
     grab_n: (inout this, in n: int, out r: std::string) -> bool = 
     {
-        if pos + n as size_t <= regex..size() {
-            r = regex..substr(pos, n as size_t);
-            pos += (n as size_t) - 1;
+        if pos + cpp2::unsafe_narrow<size_t>(n) <= regex..size() {
```

Comment:
In Cpp2 code you can use just `unsafe_narrow` these days. Cppfront will add the `cpp2::` when lowering.

Reply:
OK, will fix that.

Reply:
Fixed!